webapp/src/ts/services/enketo.service.ts

```diff
@@ -671,7 +678,7 @@ export class EnketoService {
   }
 
   private setupNavButtons($wrapper, currentIndex) {
-    if(this.currentForm.pages) {
+    if(this.currentForm?.pages) {
```

Comment:
I am a little curious about this change.  It seems like under normal circumstances `this.currentForm` should always be set (or an error thrown) since the `getEnketoForm` function will [always return an object](https://github.com/medic/cht-core/blob/c967023ddd6c651db39a9db3f3606f685a95db34/webapp/src/ts/services/enketo.service.ts#L269).

Was this change required just because of how the tests are mocking stuff or am I missing something here?

Reply:
yes, because of the tests, I haven't (don't remember) if this can be undefined at some point.